classifier.py

```diff
@@ -3629,10 +3648,9 @@ def target_age(self, audience):
         if target_age_max is None:
             target_age_max = target_age_min
 
-        # If min and max got mixed up somehow, un-mix them. This should
-        # never happen, but we fix it just in case.
+        # Err on the side of setting the minimum age too high.
         if target_age_min > target_age_max:
-            target_age_min, target_age_max = target_age_max, target_age_min
+            target_age_min = target_age_max
```

Comment:
is this backwards? I think you want both to be the higher one, which is min in this case